CONTRIBUTING.md

```diff
@@ -24,8 +24,11 @@ AMS uses a subset of [StandardIssueLabels](https://github.com/wagenet/StandardIs
 2. Create your feature branch (`git checkout -b my-new-feature`)
 3. Write tests for your feature, or regression tests highlighting a bug
 4. Write the feature itself, or fix your bug
-5. Commit your changes (`git commit -am 'Add some feature'`)
-6. Push to the branch (`git push origin my-new-feature`)
-7. Create a new Pull Request
+5. Update [CHANGELOG.md](https://github.com/rails-api/active_model_serializers/blob/master/CHANGELOG.md)
```

Comment:
Usually, the following happens:
 1. Create your branch and commit the feature/tests, push and issue PR
 2. Iterate on feedback given by the community (fix syntax, modify bits of code, add tests), pushing the new commits to the PR each time
 3. When it looks ready to merge, update the changelog + squash the commits